src/board.c

```diff
@@ -277,17 +231,14 @@ void Board_LED_Toggle(uint8_t led_gpio, uint8_t led_pin) {
 #endif
 }
 
-void Board_Headroom_Init(void){
-#ifndef TEST_HARDWARE
-    Chip_IOCON_PinMuxSet(LPC_IOCON, IOCON_HEADROOM, IOCON_FUNC1);
-    Chip_GPIO_SetPinDIROutput(LPC_GPIO, HEADROOM);
-#endif
-}
-
 void Board_Headroom_Toggle(void){
 #ifndef TEST_HARDWARE
-    // Chip_GPIO_SetPinState(LPC_GPIO, HEADROOM, 1 - Chip_GPIO_GetPinState(LPC_GPIO, HEADROOM));
-#endif
+#ifdef FSAE_DRIVERS
+    // Do nothing here, we don't care
+#else // FSAE_DRIVERS
+    Chip_GPIO_SetPinState(LPC_GPIO, HEADROOM, 1 - Chip_GPIO_GetPinState(LPC_GPIO, HEADROOM));
```

Comment:
@skoppula does this need to be commented out or can it be left in since this is an evt-only pin?